regression/sync_repository.sh

```diff
@@ -24,37 +24,33 @@
 #    all pull requests.
 
 target="`uname -n | sed -e s/[.].*//`"
+verbose=off
 
 # Locate the directory that this script is located in:
 scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 # All output will be saved to this log file.  This is also the lockfile for flock.
 logdir="$( cd $scriptdir/../../logs && pwd )"
 timestamp=`date +%Y%m%d-%H%M`
-echo "target = $target"
-echo "timestamp = $timestamp"
 logfile=$logdir/sync_repository_${target}_${timestamp}.log
 lockfile=/var/tmp/sync_repository_$target.lock
 
-case ${target} in
-  sn-fey*)
-    echo "looking for locks..."
-    echo "   FLOCKER = ${FLOCKER}"
-    echo "   lockfile will be $lockfile"
-    echo "   logfile will be $logfile"
-    echo "   ${0}"
-    echo "   $@"
-    ;;
-esac
+if [[ ${verbose:-off} == "on" ]]; then
+  echo "looking for locks..."
+  echo "   FLOCKER     = ${FLOCKER}"
+  echo "   lockfile    = $lockfile"
+  echo "   logfile     = $logfile"
+  echo "   script name = ${0}"
+  echo "   script args = $@"
+fi
 
 # Prevent multiple copies of this script from running at the same time:
 [ "${FLOCKER}" != "${lockfile}" ] && exec env FLOCKER="${lockfile}" flock -en "${lockfile}" "${0}" "$@" || :
 
```

Comment:
Instead of evaluating to a no-op, would it be useful to set an alternate `verbose` variable?
When `verbose` and the alternate are both "on", something like "another copy of ${0} is running..." could be echoed.
Also, is `verbose` set manually in this script for testing only?